### PR TITLE
Issue 1: Default global content styling

### LINKED ISSUE #1: Default global content styling
- [x] Adjust width of body for site pages as on original site.
- [x] Adjust default content styles (default colors, font sizes etc.) in styles.css
- [x] Add fallback font
- [x] Add favicon
- [x] Adjust media breakpoints, as/if required

### PR DESCRIPTION
**Pull request by msagolj**
- [x] Adjust width of body for site pages as on original site.
- [x] Adjust default content styles (default colors, font sizes etc.) in styles.css
- [x] Add fallback font
- [x] Add favicon
- [x] Adjust media breakpoints, as/if required 

Fix #1 

Test URLs:
- Before:
https://main--piramal--aemsites.hlx.page
https://main--piramal--aemsites.hlx.page/mirko-test-page-styles
- After: 
https://issue-1--piramal--aemsites.hlx.page
https://issue-1--piramal--aemsites.hlx.page/mirko-test-page-styles

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -212,19 +277,32 @@ main img {
...
+/* max width for all wrapper divs */
+.section > div {
+  max-width: var(--max-width);
+  margin: auto;
+}
+
+@media (width >= 900px) {
+  /* max width for all wrapper divs */
+  :root {
+    --max-width: 700px;
```
Comment (asthabh23) [inline styles/styles.css:294]: wouldn't `max-width` setting be sufficient for  just `width>=1200`? (can be removed from tablet and mobile ones)
so as we reduce the browser window width, we can handle maybe via padding and margin...

**Review by asthabh23 — COMMENTED** [review]
Just one comment for the `max-width` setting (good to test), rest looks good.

**Comment by asthabh23** [comment]
Also, @msagolj 
as per : https://pagespeed.web.dev/analysis/https-issue-1--piramal--aemsites-hlx-page/2oootukycw?utm_source=psi&utm_medium=redirect&form_factor=mobile

there is an accessibility issue coming up around the sequence of headings (h3 -> h1 -> h4) - so levels are getting skipped.
This needs to be updated to maintain the sequence.(highlighted parts in DOM below)
image

**Comment by msagolj** [comment]
> Also, @msagolj as per : https://pagespeed.web.dev/analysis/https-issue-1--piramal--aemsites-hlx-page/2oootukycw?utm_source=psi&utm_medium=redirect&form_factor=mobile
> 
> there is an accessibility issue coming up around the sequence of headings (h3 -> h1 -> h4) - so levels are getting skipped. This needs to be updated to maintain the sequence.(highlighted parts in DOM below) image

@asthabh23 these are all headings where H level can be edited by the author. h3 -> h1 should be ok, h1 to h4 are titles so they can be changed. in their original site they dont see the issue because because the h4 is text so that on them to decide what supposed to be a title or just text.

In general their site is inconsistent whith using <div> or <hx> for titles e.g. 'Happy Customers ' is an H2 while 'Primal Since 1980' is a styled DIV

**Review by asthabh23 — APPROVED** [review]

**Pull request merged**